### ISSUE TITLE
Vary the content of the authentication email that is sent to the user

### ISSUE THREAD

**Issue by jonmsawyer** [Kromey/fbxnano#23]
In passwordless authentication, the email sent has no varying content. This makes conversation-based email hosts such as Gmail render the new email as an unchanged quote. See **Screen Shot 1**.
#### Screen Shot 1

> ![email1](https://cloud.githubusercontent.com/assets/1679983/19201650/2f9334d2-8c7b-11e6-9dae-09c7e23b13ee.jpg)

In Gmail, in order to log in more than once in a day, I have to click the little ellipsis [ ... ] icon to expand my new email. See **Screen Shot 2**.
#### Screen Shot 2

> ![email2](https://cloud.githubusercontent.com/assets/1679983/19201649/2f917ba6-8c7b-11e6-9ea9-f262bbb998c6.jpg)
## Proposal

Simply change the text of the email. Here's one template that I think would work well (also added is a new paragraph at the end of the template):
#### email.html

``` html
<html>
<body>
<p>
Hi {{ username }},
</p>
<p>
On {{ human_readable_datetime }}, you (or someone) requested an email to log you into the
Alaska Writers website. This is that email!
</p>
<p>
To log in, simply use this link:
<a href="{{ authn_url }}">{{ authn_url }}</a>
</p>
<p>
If you did not initiate this request, you may disregard this email.
</p>
</body>
</html>
```
#### email.txt

```
Hi {{ username }},

On {{ human_readable_datetime }}, you (or someone) requested an email to log you into the
Alaska Writers website. This is that email!

To log in, simply use this link:
{{ authn_url }}

If you did not initiate this request, you may disregard this email.
```

**Comment by Kromey** [comment]
Users can easily get around this by deleting the email after they receive it, which isn't to say that this isn't a valid improvement. However, varying only the first paragraph may still result in e.g. Gmail hiding the rest behind that danged ellipsis, so it may require a varying footer paragraph instead/in addition.

**Comment by jonmsawyer** [comment]
It's not just the first paragraph, it's also the second to last. I replaced `Log in now!` with `{{ authn_url }}` since the token used to log into the site is always different.

**Comment by Kromey** [comment]
Fixed in c1b4e2f
